docs/cellery-syntax.md

```diff
@@ -117,11 +119,33 @@ public function build(cellery:ImageName iName) returns error? {
 ```
 
 ###### Expose
-An `HttpApiIngress` can be exposed as an API by setting `expose` field. This field accepts two values.
+A particular `HttpApiIngress` can be exposed as an API by setting `expose` field. This field accepts two values.
 
     -  `local`: Expose an HTTP API via local cell gateway.
     -  `global`: Expose an HTTP API via global gateway.
 
+By setting `expose` to `global`, the API will be published to the global gateway which will expose the API to external 
+traffic. In this case, the cell name and the context provided in the ingress is combined to create the full context for the 
+global API.
+
+In addition to exposing HTTP ingresses selectively, users can expose all the ingresses of a particular cell to outside by 
+using the `globalPublisher` configuration. 
+
+```ballerina
+cellery:CellImage hrCell = {
+    globalPublisher: {
```

Comment:
I remember, we had discussion to provide a struct for expose attribute in the component's ingress to specify how the global API will be published. But this looks bit different. Also, how can we publish all ingresses in all components under same context ?